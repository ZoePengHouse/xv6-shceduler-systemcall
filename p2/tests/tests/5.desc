Test negative buffer size handling
