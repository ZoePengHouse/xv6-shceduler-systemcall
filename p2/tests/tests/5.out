XV6_TEST_OUTPUT Negative buffer length handled correctly.
